client/service/src/builder.rs

```diff
@@ -1011,18 +1007,34 @@ ServiceBuilder<
 			);
 		}
 
-		// Prometheus metrics
+		// Prometheus metrics.
 		let metrics = if let Some(PrometheusConfig { port, registry }) = config.prometheus_config.clone() {
+			// Set static metrics.
+			register(Gauge::<U64>::with_opts(
+				Opts::new(
+					"build_info",
+					"A metric with a constant '1' value labeled by name, version, and commit."
+				)
```

Comment:
If you remove one tab here, it will not be look that weird.

(All the other ones need to be moved to the left as well)

Reply:
If I remove one tab in 1017 it would not align with the opening bracket in 1014. I am not sure I understand your suggestion @bkchr.